### ISSUE TITLE
[Feb 2025] Accessibility Audit: Static site How it works section

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8510]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the pages in the "How it works" section of the static site that have been updated in the last 3 months. Do the page scan as well as any relevant guided tests.
Pages to visit:
- [x] https://dev3.simplereport.gov/using-simplereport/
- [x] https://dev3.simplereport.gov/using-simplereport/select-your-testing-facility/
- [x] https://dev3.simplereport.gov/using-simplereport/report-test-results/conduct-and-submit-tests/
- [x] https://dev3.simplereport.gov/using-simplereport/report-test-results/bulk-upload-results/
- [x] https://dev3.simplereport.gov/using-simplereport/report-test-results/backdate-a-test/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-results/review-test-results/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-results/print-someones-test-results/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-results/download-test-results/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-results/correct-a-previous-test-result/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-results/see-upload-history/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-people-you-test/add-a-new-patient/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-people-you-test/bulk-upload-patients/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-people-you-test/self-registration/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-people-you-test/archive-a-person/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-people-you-test/update-someones-profile/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-users/invite-new-users/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-users/manage-user-permissions/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-users/edit-user-information/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-users/reset-user-password/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-users/reset-user-mfa/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-users/reactivate-users/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-facility-info/add-a-facility/
- [x] https://dev3.simplereport.gov/using-simplereport/manage-facility-info/update-facility-settings/
- [x] https://dev3.simplereport.gov/using-simplereport/test-for-other-diseases/hiv/
- [x] https://dev3.simplereport.gov/using-simplereport/test-for-other-diseases/influenza/
- [x] https://dev3.simplereport.gov/using-simplereport/test-for-other-diseases/rsv/
- [x] https://dev3.simplereport.gov/using-simplereport/reset-your-simplereport-password/
- [x] https://dev3.simplereport.gov/using-simplereport/whats-new/

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See ticket from previous audit #5767, #6336

**Comment by emyl3** [comment]
`/using-simplereport` - [202502-dev.simplereport.gov_using-simplereport.json](https://github.com/user-attachments/files/18971462/202502-dev.simplereport.gov_using-simplereport.json)
